src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const sourceStringSplit = sourceString.split(';');
+
+  for (const key of sourceStringSplit) {
```

Comment:
Pls avoid loops in this task (see checklist point 5). How about starting with `.map` and adding a few other chain methods to achieve the same?))